external/SDL/build-scripts/androidbuildlibs.sh

```
@@ -0,0 +1,85 @@
+# Useful NDK arguments:
+#
+#  NDK_DEBUG=1          - build debug version
+#  NDK_LIBS_OUT=<dest>  - specify alternate destination for installable
+#                         modules.
+#
+
+
+# Android.mk is in srcdir
+srcdir=`dirname $0`/..
+srcdir=`cd $srcdir && pwd`
+cd $srcdir
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)